user/user.go

```diff
@@ -80,6 +80,7 @@ func SearchOtherUsers(w http.ResponseWriter, r *http.Request) {
 	query := params["query"]
 	filter := bson.M{
 		"$or": []bson.M{
+			{"deactivated": false},
```

Comment:
I do not think we should do this, since we are changing the implementation. Someone else is working on moving it to member level and it is possible to search for deactivated member